raster-foundry/raster-foundry
# Mask mosaic tile layers

## Overview

This PR includes changes to get mosaic tile layers masked. It doesn't work right now because my changes to MAML were insufficient to get masking applied to multiband tiles, though you can get the first band of a multiband tile masked, so that's sort of cool.

### Checklist


### Demo

![mask](https://user-images.githubusercontent.com/5702984/56159267-404a8400-5f92-11e9-8d90-065adc8937c9.gif)

### Notes

~[wip] until masking pr in maml can be merged -- geotrellis/maml#80~

## Testing Instructions

 * add an aoi to a project layer
 * view tiles
 * ch-ch-ch-check it out -- your mask should be applied to the resulting tiles

Closes #4727
